### PR TITLE
adds rgi version 3.0.9

### PR DESCRIPTION
**Pull request by raphenya**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by bgruening** [comment]
zlib is missing, please look at https://github.com/bioconda/bioconda-recipes/blob/master/GUIDELINES.md#zlib how to fix this.

File: recipes/rgi/build.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# copy MetaGeneMark key to working directory
+cp ./package/rgi/mgm/gm_key ~/.gm_key
```
Comment (bgruening) [inline recipes/rgi/build.sh]: This does not work or does not have any influence. You are copying the file during build-time and it is not on the user system as run-time.
Is this step really needed?

**Comment by bgruening** [comment]
ping @raphenya

File: recipes/rgi/build.sh (hunk 1 above)
Comment (raphenya) [inline recipes/rgi/build.sh]: I'm working on replacing the MetaGeneMark tool

**Comment by raphenya** [comment]
@bgruening I want to add prodigal dependencies for rgi recipes. I'm getting the following error:

```
Fixing permissions
BUILD END: rgi-3.1.0-py_0
TEST START: rgi-3.1.0-py_0
Fetching package metadata .........
Solving package specifications: .
Error: Packages/dependencies missing in current osx-64 channels: 
  - rgi 3.1.0 py_0 -> prodigal
  - prodigal
```
my meta.yaml file looks like so for the requirements :

```
requirements:
  build:
    - python
    - argparse
    - biopython
    - blast # [linux]
    - six >=1.7.0
    - gcc   # [not osx]
    - llvm  # [osx]
    - zlib
  run:
    - python
    - argparse
    - blast # [linux]
    - biopython
    - six >=1.7.0
    - libgcc    # [not osx]
    - zlib
    - prodigal
```

**Comment by bgruening** [comment]
Have you added `-c bioconda` it seems that prodigal is available for osx: https://anaconda.org/bioconda/prodigal

**Comment by raphenya** [comment]
Yes, I have installed the prodigal on my mac. Running the build gives me that error, even if I remove prodigal.

**Comment by raphenya** [comment]
shoud I use ```- prodigal # [not osx] ```?

**Comment by bgruening** [comment]
This should work :(
Have you used `-c bioconda` in your `conda build` step?
Can you submit your latest changes?

File: recipes/rgi/meta.yaml [hunk 2]
```diff
@@ -32,6 +32,7 @@ requirements:
     - six >=1.7.0
     - libgcc    # [not osx]
     - zlib
+    - prodigal  # [not osx]
```
Comment (bgruening) [inline recipes/rgi/meta.yaml]: please remove this, we will see what happens and try to fix prodigal

**Comment by raphenya** [comment]
I used this command to build:

```conda build recipes/rgi/ --channel bioconda```

and the build went ok

**Comment by bgruening** [comment]
`Error: MD5 mismatch: '68c7f04705b9975cda7e757bae9f649e' != '11ce21313834d23380d9742583e23cca'`

**Comment by bgruening** [comment]
Awesome! Thanks @raphenya!

**Comment by raphenya** [comment]
solve the following error:
 ```
Error: MD5 mismatch: '68c7f04705b9975cda7e757bae9f649e' != '11ce21313834d23380d9742583e23cca'
```

by removing old files from cache e.g `rm /../../anaconda2/conda-bld/src_cache/*`

**Pull request merged**